### PR TITLE
feat: Added PHP Request Examples for NotificationApi in API Reference Documentation

### LINKED ISSUE #282: Add PHP Examples for NotificationApi in API Reference Documentation
path: /api-reference/notification/get-notifications

the current examples are auto generated by [mintlify ](https://mintlify.com/blog/autogenerate-openapi)

Here is the example I would like to add:
image

### PR DESCRIPTION
**Pull request by nabinbhatt**
closes #282

### CODE REVIEW

File: api-reference/notification/get-notification-graph-statistics.mdx [hunk 1]
```diff
@@ -23,4 +23,14 @@ notificationApi = NotificationApi(url, api_key)
...
 list(graphStats)
 ```
+
+```php PHP
+use Novu\SDK\Novu;
+
+$novu = new Novu('<NOVU_API_KEY>');
+
+$notificationGraphStats = $novu->getNotificationGraphStats()->toArray();
+echo json_encode($notificationGraphStats);
```
Comment (unicodeveloper) [inline api-reference/notification/get-notification-graph-statistics.mdx:33]: Please remove this line. Not needed.

File: api-reference/notification/get-notification-statistics.mdx [hunk 2]
```diff
@@ -34,4 +34,13 @@ notificationApi = NotificationApi(url, api_key)
 notification_stats = notificationApi.stats()
 ```
 
+```php PHP
+use Novu\SDK\Novu;
+
+$novu = new Novu('<NOVU_API_KEY>');
+
+$notificationStats = $novu->getNotificationStats()->toArray();
+echo json_encode($notificationStats);
```
Comment (unicodeveloper) [inline api-reference/notification/get-notification-statistics.mdx:43]: Please remove this line. Not needed.

File: api-reference/notification/get-notification.mdx [hunk 3]
```diff
@@ -35,4 +35,13 @@ notification = notificationApi.get("<notificationId>")
 print(notification)
 ```
 
+```php PHP
+use Novu\SDK\Novu;
+
+$novu = new Novu('<NOVU_API_KEY>');
+
+$notification = $novu->getNotification($notificationId)-toArray();
+echo json_encode($notification)
```
Comment (unicodeveloper) [inline api-reference/notification/get-notification.mdx:44]: Please remove this line. Not needed.

File: api-reference/notification/get-notifications.mdx [hunk 4]
```diff
@@ -36,4 +36,13 @@ notificationApi = NotificationApi(url, api_key)
 notifications = notificationApi.list()
 ```
 
+```php PHP
+use Novu\SDK\Novu;
+
+$novu = new Novu('<NOVU_API_KEY>');
+
+$notifications = $novu->getNotifications()->toArray();
+echo json_encode($notifications);
```
Comment (unicodeveloper) [inline api-reference/notification/get-notifications.mdx:45]: Please remove this line. Not needed.

**Review by unicodeveloper — COMMENTED** [review]
Nitpick: Please attend to the comments.

File: api-reference/notification/get-notification-graph-statistics.mdx (hunk 1 above)
Comment (nabinbhatt) [reply api-reference/notification/get-notification-graph-statistics.mdx:33]: Should I remove the echo statement?

File: api-reference/notification/get-notification-statistics.mdx (hunk 2 above)
Comment (nabinbhatt) [reply api-reference/notification/get-notification-statistics.mdx:43]: don

File: api-reference/notification/get-notification-graph-statistics.mdx (hunk 1 above)
Comment (unicodeveloper) [reply api-reference/notification/get-notification-graph-statistics.mdx:33]: Yes

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**